backend/routes/api.php

```diff
@@ -15,8 +16,24 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
+// Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
+//     return $request->user();
+// });
+
+// ADMIN ROUTE
+Route::post('admin/login', [AdminController::class, 'login']);
+
+Route::middleware(['auth:sanctum', 'abilities:admin'])->prefix('admin')->group(function () {
+	Route::get('details', [AdminController::class, 'userDetails']);
```

Comment:
You dont need details for admin

Reply:
That's only for testing the token sir 🙇
I will remove it right away.